plugins/modules/notification/slack.py

```diff
@@ -85,12 +85,6 @@
     choices:
       - 'full'
       - 'none'
-  validate_certs:
-    description:
-      - If C(no), SSL certificates will not be validated. This should only be used
-        on personally controlled sites using self-signed certificates.
-    type: bool
-    default: 'yes'
```

Comment:
You cannot simply remove options. That breaks backwards compatibility for people using this option. Also, this option **is** used: https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/urls.py#L1593

Reply:
Gotcha – thanks for clearing that up – just didn't realize it could/would be passed all the way through.